README.md

```diff
@@ -1,2 +1,126 @@
+```
+
+Add ${HOME}/.local/bin to your PATH environment variable
+
+By default, Minikube uses the kvm driver with 6,000 MB of memory. This is not enough to run all the services, so we need to increase the allocated memory. In our experience, 10 GB of memory is fine.
+
+`$ minikube config set memory 10240`
+
+Then, the following command will download a qcow2 image of Minikube and start a virtual machine from it. It will then wait for the Kubernetes API to be ready.
+
+`$ minikube start`
+
+We need to enable the dashboard, ingress and olm addons. The dashboard addon installs the dashboard service that exposes the Kubernetes objects in a user interface. The ingress addon allows us to create Ingress CRs to expose the Tackle UI and Tackle Hub API. The olm addon allows us to use an operator to deploy Tackle.
+
+`$ minikube addons enable dashboard`
+`$ minikube addons enable ingress`
+`$ minikube addons enable olm`
+
+
+The following command gives us the IP address assigned to the virtual machine created by Minikube.
+It's used when interacting with tackle UI image installed on the minikube cluster.
+
+```
+$ minikube ip
+192.168.39.23
+```
+
+### Configuring kubectl (optional)
+Minikube allows us to use the kubectl command with `minikube kubectl. To make the experience more Kubernetes-like, we can set a shell alias to simply use kubectl.
+The following example shows how to do it for Bash on Fedora 35.
+
+```
+$ mkdir -p ~/.bashrc.d
+$ cat << EOF > ~/.bashrc.d/minikube
+alias kubectl="minikube kubectl --"
+EOF
+$ source ~/.bashrc
+```
+
+### Accessing the Kubernetes dashboard
+We may need to access the dashboard, either simply to see what's happening under the hood, or to troubleshoot an issue. We have already enabled the dashboard addon in a previous command.
+
+We can use the kubectl proxy command to enable that. The following command sets up the proxy to listen on any network interface (useful for remote access), on the 18080/tcp port (easy to remember), with requests filtering disabled (less secure, but necessary).
+
+`$ kubectl proxy --address=0.0.0.0 --port 18080 --disable-filter=true`
+
+We can now access the minikube dashboard through the proxy.
+In the following URL, replace the IP address with your workstation IP address. 
+
+http://192.168.0.1:18080/api/v1/namespaces/kubernetes-dashboard/services/http:kubernetes-dashboard:/proxy/#/
+
+
+### Installing Tackle on Minikube or Kubernetes
+
+Follow Tackle2 documentation [https://github.com/konveyor/tackle2-operator/blob/main/docs/k8s.md](k8s.md) to install Tackle on Minikube or Kubernetes.
+
+Then deploy Tackle by running: 
+`kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml`
+
+Wait few minutes (otherwise the following will fail) then launch Tackle by applying the following CR:
+
+```
+cat << EOF | kubectl apply -f -
+kind: Tackle
+apiVersion: tackle.konveyor.io/v1alpha1
+metadata:
+  name: tackle
+  namespace: konveyor-tackle
+spec:
+EOF
+```
```

Comment:
Might be good to include this command for waiting for pods to spin up before running proxy. 

```
kubectl wait deployment tackle-keycloak-sso \                                                                                                                                                                                                   
    -n konveyor-tackle --for condition=Available --timeout=5m

```

Reply:
Good point, this one is really handy.